app/views/admin/sites/edit.html.erb

```diff
@@ -4,22 +4,20 @@
   <div class="grid-column">
     <%= render "admin/shared/site_tabs" %>
   </div>
-
-  <div class="column-two-thirds extra-gutter">
-    <% if params[:tab] == "gaelic" %>
-      <%= render "form", tab: "gaelic" %>
-    <% elsif params[:tab] == "petitions" %>
-      <%= render "form", tab: "petitions" %>
-    <% elsif params[:tab] == "moderation" %>
-      <%= render "form", tab: "moderation" %>
-    <% elsif params[:tab] == "feedback" %>
-      <%= render "form", tab: "feedback" %>
-    <% elsif params[:tab] == "access" %>
-      <%= render "form", tab: "access" %>
-    <% elsif params[:tab] == "features" %>
-      <%= render "form", tab: "features" %>
-    <% else %>
-      <%= render "form", tab: "english" %>
-    <% end %>
-  </div>
 </div>
+
+<% if params[:tab] == "gaelic" %>
```

Comment:
maybe move this to a controller `current_tab` helper method of some kind?

```rb
def current_tab
  @current_tab = params[:tab] || 'english'
end
```

?

Reply:
Will do later this week